broker/plugins/chronos/plugin.py

```diff
@@ -44,13 +45,14 @@ def start_application(self, data):
 
             chronos = ManagerChronos(url, user, password)
 
-            # Obtain arguments 
-            start_job = time.time()
-            job = data['info_plugin']['job']            
-            deadline = data['info_plugin']['qos']['deadline'] #+ start_job 
-            job_duration = data['info_plugin']['qos']['duration'] # seconds
-            desv_deadline = data['info_plugin']['qos']['desv_deadline'] # %/100 (e.g 5% --> desv_deadline=0.05 )
-            jobname = job['name'] 
+            # Obtain arguments
+            # start_job = time.time()
```

Comment:
Again, please remove commented code but leave useful comments like in line 48

Reply:
Done!